File: data/solutions/P6/solution.cpp
#include <iostream>
using namespace std;

int main() {
    int t;
    cin >> t;  // read number of testcases
    
    while (t--) {
        long long n;
        cin >> n;  // read the number
        
        if (n % 2 == 0) {
            cout << "EVEN" << endl;  // n divisible by 2 → EVEN
        } else {
            cout << "ODD" << endl;   // n not divisible by 2 → ODD
        }
    }
    return 0;
}

File: data/solutions/P6/solution.py
t = int(input())  # read number of testcases
for _ in range(t):
    n = int(input())  # read the number
    if n % 2 == 0:
        print("EVEN")  # divisible by 2 → EVEN
    else:
        print("ODD")   # not divisible by 2 → ODD
